composer.json

```diff
@@ -15,14 +15,15 @@
         "php-http/message": "^1.6",
         "php-http/message-factory": "^1.0",
         "phpro/soap-client": "^0.7.11",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0",
+        "symfony/http-foundation": "^3.4"
     },
     "require-dev": {
         "cweagans/composer-patches": "^1.6",
-        "openeuropa/code-review": "dev-master",
-        "php-http/mock-client": "^1.0",
+        "openeuropa/code-review": "~1.0@beta",
```

Comment:
This needs to stay at `dev-master` as we are already using the next version of the code review library on this project to help us fix the big code generation we have to endure here.

Reply:
Done.